### DISCUSSION TITLE
User Song History

### DISCUSSION THREAD

**Discussion by mathieugoodman** [vicwomg/pikaraoke#209] [Ideas]
Here's yet another idea: it would be cool if a named user could see the history of the songs they queued/sang in past sessions, and allow adding songs from that history list back to the queue in new sessions.

When I do karaoke, I always try to remember what I enjoyed singing in the past, but that's not always easy depending on how long ago it was or how sober I was 😉 

Thanks!

**Comment by poeticize** [comment]
Check out the PR https://github.com/vicwomg/pikaraoke/pull/790 I just submitted. yesterday  It does all this  to make it easy for the KJ and allows you to save out and re-import all user histories to back that bit up.
